static/app/views/codecov/tests/preOnboarding.tsx

```diff
@@ -0,0 +1,207 @@
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import testsAnalyticsSummary from 'sentry-images/features/test-analytics-summary.svg';
+
+import {LinkButton} from 'sentry/components/core/button';
+import Link from 'sentry/components/links/link';
+import {IconGithub} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+const INSTRUCTIONS_TEXT = {
+  noOrgs: {
+    header: t('Get Started by installing the GitHub Sentry App'),
+    subtext: t(
+      "You need to install the Sentry App on your GitHub organization as an admin. If you're not an admin, you will need to make sure your GitHub organization admins approve the installation of the Sentry App on GitHub."
+    ),
+    mainCTA: t('Add installation'),
+    mainCTALink: 'https://github.com/apps/sentry-io',
+  },
+  hasOrgs: {
+    header: t('Request Updated Permissions on your Integrated Organization'),
+    subtext: t(
+      "Sentry is requesting updated permissions access to your integrated organization(s). Admin required. If you're not an admin, reach out to your organization's owner to update the Sentry app permissions."
+    ),
+    mainCTA: t('Review permissions'),
+    // TODO: can we get the xxx installation id? it is unique to each signed in GH user
+    mainCTALink: '/settings/installations/xxxx/permissions/update',
+  },
+};
+
+// TODO: this should come from the backend
+const HAS_INTEGRATED_ORGS = false;
+
+export default function TestPreOnboardingPage() {
+  const instructionSet = HAS_INTEGRATED_ORGS
+    ? INSTRUCTIONS_TEXT.hasOrgs
+    : INSTRUCTIONS_TEXT.noOrgs;
+  return (
+    <LayoutGap>
+      <IntroSection>
+        <img src={testsAnalyticsSummary} />
+        <div>
+          <h2>{t('Say Goodbye to Flaky Tests')}</h2>
+          <StyledP
+            css={css`
+              margin-bottom: ${space(1)};
+            `}
+          >
+            {t(
+              'Test Analytics offers data on test run times, failure rates, and identifies flaky tests to help decrease the risk of deployment failures and make it easier to ship new features quickly. '
+            )}
+            <Link to="https://docs.codecov.com/docs/test-analytics">
+              {t('Learn more')}
+            </Link>
+          </StyledP>
+          <ItalicP>
+            {t(
+              'No more staring at failing tests wondering, “Is it me, or is it just flaky?”'
+            )}
+          </ItalicP>
+        </div>
+      </IntroSection>
+      <InstructionsSection>
+        <h2>{instructionSet.header}</h2>
+        <p>{instructionSet.subtext}</p>
+        <ButtonBar>
+          <LinkButton priority="primary" href={instructionSet.mainCTA}>
+            {instructionSet.mainCTA}
+          </LinkButton>
+          <LinkButton priority="default" href="/settings/integrations/github">
+            Learn more
+          </LinkButton>
+        </ButtonBar>
+        <PrerequisitesSection>
```

Comment:
Since we're defaulting to GH for now this is probably fine but it might be worth keeping this in mind for any provider. I think it's fine cause if we ever support other providers, we'll have a different markup anyway so that will be the time to expand this code